Apps/Sandcastle/gallery/3D Tiles.html

```diff
@@ -330,329 +272,6 @@
-    console.log('New dynamic SSE factor: ' + tileset.dynamicScreenSpaceErrorFactor);
-});
-
-Sandcastle.addToolbarButton('Trim tiles (cache)', function() {
-    tileset.trimLoadedTiles();
-});
-
-Sandcastle.addToolbarButton('Picking on/off', function() {
-    pickingEnabled = !pickingEnabled;
-});
-
-// Styling ////////////////////////////////////////////////////////////////////
-
-var styleElements = [];
-var numberofColors = 6;
-var currentPropertyName = 'Height';
-
-function getRandomColor() {
-    var color = Cesium.Color.fromRandom();
-    color.alpha = 1.0;
-    return color.toCssColorString();
-}
-
-function styleFunction(name) {
-    var conditions = [];
-    var intervalSize = Math.floor(100/numberofColors);
-    for (var i = numberofColors; i >= 0; --i) {
-        var cond = '${' + name + '} > ' + (i * intervalSize);
-        conditions.push([cond, getRandomColor()]);
-    }
-    conditions.push(['true', getRandomColor()]);
-
-    tileset.style = new Cesium.Cesium3DTileStyle({
-        color : {
-            conditions : conditions
-        },
-        show : '${' + name + '} >= 0'
-    });
-
-    currentPropertyName = name;
-}
-
-function getStyleFunction(name) {
-    return function() {
-        styleFunction(name);
-    };
-}
-
-function addStyleUI() {
-    var button;
-
-    var properties = tileset.properties;
-    if (Cesium.defined(properties)) {
-        for (var name in properties) {
-            // Include all properties except longitude and latitude
-            if (properties.hasOwnProperty(name) && (name !== 'Longitude') && (name !== 'Latitude')) {
-                button = Sandcastle.addToolbarButton('Style with ' + name, getStyleFunction(name));
-                styleElements.push(button);
-            }
-        }
-    }
-
-    button = Sandcastle.addToolbarButton('Custom style', function() {
-        var style = new Cesium.Cesium3DTileStyle();
-        style.show = {
-            evaluate : function(frameState, feature) {
-                tileset.makeStyleDirty(); // Force the style to be constantly evaluated
-                return Cesium.Math.nextRandomNumber() > 0.5;
-            }
-        };
-        style.color = {
-            evaluateColor : function(frameState, feature, result) {
-                tileset.makeStyleDirty(); // Force the style to be constantly evaluated
-                return Cesium.Color.fromRandom(undefined, result);
-            }
-        };
-        tileset.style = style;
-    });
-    styleElements.push(button);
-
-    button = Sandcastle.addToolbarButton('Remove style', function() {
-        tileset.style = undefined;
-    });
-    styleElements.push(button);
-
-    function setColorBlendMode(colorBlendMode) {
-        return function() {
-            tileset.colorBlendMode = colorBlendMode;
-        };
-    }
-
-    var menu = Sandcastle.addToolbarMenu([{
-        text : 'Color Blend Mode'
```

Comment:
This is probably good for now. One suggestion I have is for the inspector to return the tileset's properties so it is easy to know what variables to use to create styles.